.github/workflows/update-versions-and-files.yml

```diff
@@ -28,17 +28,23 @@ jobs:
           OSC_PASSWORD: ${{ secrets.OSC_PASSWORD }}
           OSC_USER: "defolos"
 
-      - name: create a pull request with the updated versions
+      # update the files **after** we updated the versions to get the new
+      # mariadb files
```

Comment:
this comment is hard to parse. how about

```suggestion
# update versions first so that updating the files  can fetch based on the updated versions
```